### PR TITLE
Fix spider for Tim Hortons

### PR DESCRIPTION
**Pull request by nbeecher**
Fix for Tim Hortons Spider

### CODE REVIEW

File: locations/spiders/timhortons.py [hunk 1]
```diff
@@ -2,52 +2,67 @@
 import re
 from locations.items import GeojsonPointItem
 
-class TimhortonsSpider(scrapy.Spider):
 
+class TimhortonsSpider(scrapy.Spider):
     name = "timhortons"
     allowed_domains = ["locations.timhortons.com"]
+    custom_settings = {
+        'DUPEFILTER_CLASS': 'scrapy.dupefilters.BaseDupeFilter',
```
Comment (iandees) [inline locations/spiders/timhortons.py]: Why did you need to add this?

Comment (nbeecher) [reply locations/spiders/timhortons.py]: I had to hit parse_city_stores sometimes twice (in def parse_city:) depending if there were multiple stores for one URL so I had to do a try except to figure out if the url contained 1 store or multiple.  Scrapy by default does not allow you to send a request on the same URL twice. The original scraper assumed single store/multiple stores based on regex of the URL but that was incorrect.

**Comment by iandees** [comment]
Thank you for doing this! While I was testing your changes, it looked like they're using the same company as several others, so it was an easy copy/paste from an existing spider. I did that in https://github.com/alltheplaces/alltheplaces/pull/1199.
